src/posts/lucky_0.26.cr

```diff
@@ -0,0 +1,205 @@
+    Or in many cases, it's becoming more common practice to require a user to type something to "confirm" they really do want to delete something.
+    For these use cases, we now have the DeleteOperation.
+
+    Every model has a DeleteOperation that you can inherit from the same as the SaveOperation. Here's an example:
+
+    ```crystal
+    # src/operations/delete_repo.cr
+    class DeleteRepo < Repo::DeleteOperation
+      attribute confirm_delete : String
+
+      before_delete do
+        if confirm_delete.value != "luckyframework/\#{git_name.value}"
+          confirm_delete.add_error("You must confirm the delete")
+        end
+      end
+
+      after_delete do |deleted_repo|
+        CacheSweeper.run!("luckyframework/\#{deleted_repo.git_name}")
+      end
+    end
+
+    # src/actions/repos/delete.cr
+    class Repos::Delete < BrowserAction
+      delete "/repos/:repo_id" do
+        repo = RepoQuery.find(repo_id)
+
+        DeleteRepo.destroy(repo) do |operation, deleted_repo|
+          if operation.deleted?
+            flash.success = "The repo \#{deleted_repo.git_name} has been deleted"
+            redirect to: Home::Index
+          else
+            TimeBomb.start_countdown(10.seconds)
+          end
+        end
+      end
+    end
+    ```
+
+    [Read more](https://github.com/luckyframework/avram/pull/573) on this feature.
+
+    ### CIText Support
+
+    This was new to some of us that had never even heard of "citext" before. The "citext" column in postgres is "Case-Insensitive Text". This allows you to
+    store a string in your database, and do lookups against it without the need to do something like `"WHERE LOWER(email) = ?", email.downcase`. Instead, your
+    users could spell their email like "LucKyDawG@gmAIl.com" and still log in with "luckydawg@gmail.com".
+
+    By default, all `String` columns in Avram are case sensitive, but if you'd like to use this citext column, just add `case_sensitive: false` to your migrations!
+
+    ```crystal
+    # NOTE: You must enable the extension for this feature to work!
+    enable_extension "citext"
+
+    create table_for(User) do
+      add name : String
+      add email : String, case_sensitive: false
+    end
+    ```
+
+    [Read more](https://github.com/luckyframework/avram/pull/608) on this feature.
+
+    ### Composite Keys
+
+    Speaking of migrations... We're slowly working on adding in "composite-primary keys". This is the concept of having 2 (or more) columns that are combined for primary key
+    lookups in your database. Generally you'll just use a single `id` column, but with a composite primary key, you could have "id1" and "id2", and a record is looked up
+    using both of those fields.
+
+    For now, we've only enabled this on the migration side. [See the PR](https://github.com/luckyframework/avram/pull/616).
```

Comment:
Does this mean you can't query for it?

Reply:
I honestly have no clue how it works 😂  I debated on not even mentioning it...

Reply:
Yea, you can make the migration but you cannot actually model it with Avram yet.

Reply:
Yeah, I think I'll remove the mention of it then. It's not really useful yet.

Reply:
It's a great feature, but yeah feels a bit weird to highlight it if we only support half of it. Copy this whole section into the next release notes then it's already written!

Reply:
@jwoertink could you replace it with a section on being able to run flow specs with the browser not in headless mode? So now you don't have to use `open_screenshot` to figure out where your spec is failing since you can have the browser showing and use `flow.pause` to stop it where you want.